### PR TITLE
Update CDK version for trello and zenloop

### PR DESCRIPTION
**Pull request by girarda**
Version 0.68.4 was a breaking change.

The connectors were also migrated to the base image and to poetry

### CODE REVIEW

File: docs/integrations/sources/instatus.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ The Instatus source connector supports the following [sync modes](https://docs.a
 
 ## Changelog
 
-| Version | Date       | Pull Request                                             | Subject                    |
-|:--------|:-----------|:---------------------------------------------------------|:---------------------------|
-| 0.1.0   | 2023-04-01 | [21008](https://github.com/airbytehq/airbyte/pull/21008) | Initial (alpha) release    |
\ No newline at end of file
+| Version | Date       | Pull Request                                             | Subject                 |
+|:--------|:-----------|:---------------------------------------------------------|:------------------------|
+| 0.1.1   | 2024-03-29 | [](https://github.com/airbytehq/airbyte/pull/) | Update CDK version      |
```
Comment (natikgadzhi) [inline docs/integrations/sources/instatus.md:60]:
```suggestion
| 0.1.1   | 2024-03-29 | [36592](https://github.com/airbytehq/airbyte/pull/36592) | Update CDK version      |
```

**Review by natikgadzhi — APPROVED** [review]
Added a changelog suggestion to mention this PR. 

I'm itching to remove dockerfiles and convert all three to Poetry, since we'd have to republish anyway. Want me to do it and push up a commit to this PR?

**Comment by girarda** [comment]
@natikgadzhi sorry for late update. I'll get back to this soon. Yes, I'll also migrate to poetry while I'm at it

**Review by alafanechere — CHANGES_REQUESTED** [review]
@girarda I'm not sure the PR title matches the changes made: I don't think the CDK version was updated on these connector. The migration to poetry pinned the CDK to the previous connector version's CDK. I might be mistaken and these version pinning are indeed your original intent.

File: docs/integrations/sources/zenloop.md [hunk 2]
```diff
@@ -69,16 +69,17 @@ The Zenloop connector should not run into Zenloop API limitations under normal u
...
-| 0.1.3   | 2022-09-28 | [17304](https://github.com/airbytehq/airbyte/pull/17304) | Migrate to per-stream states                                        |
-| 0.1.2   | 2022-08-22 | [15843](https://github.com/airbytehq/airbyte/pull/15843) | Adds Properties stream                                              |
-| 0.1.1   | 2021-10-26 | [8299](https://github.com/airbytehq/airbyte/pull/8299)   | Fix missing seed files                                              |
-| 0.1.0   | 2021-10-26 | [7380](https://github.com/airbytehq/airbyte/pull/7380)   | Initial Release                                                     |
+| Version | Date       | Pull Request                                             | Subject                                                                         |
+|:--------|:-----------| :------------------------------------------------------- |:--------------------------------------------------------------------------------|
+| 0.1.11  | 2024-04-02 | [36592](https://github.com/airbytehq/airbyte/pull/36592) | Use python-connector-base image, poetry, and update CDK version                 |
```
Comment (girarda) [inline docs/integrations/sources/zenloop.md:74]:
```suggestion
| 0.1.11  | 2024-04-10 | [36592](https://github.com/airbytehq/airbyte/pull/36592) | Use python-connector-base image, poetry, and update CDK version                 |
```

File: docs/integrations/sources/trello.md [hunk 3]
```diff
@@ -74,19 +74,20 @@ The Trello connector should not run into Trello API limitations under normal usa
...
-| 0.1.3   | 2021-11-25 | [8183](https://github.com/airbytehq/airbyte/pull/8183)   | Enable specifying board ids in configuration               |
-| 0.1.2   | 2021-11-08 | [7499](https://github.com/airbytehq/airbyte/pull/7499)   | Remove base-python dependencies                            |
-| 0.1.1   | 2021-10-12 | [6968](https://github.com/airbytehq/airbyte/pull/6968)   | Add oAuth flow support                                     |
-| 0.1.0   | 2021-08-18 | [5501](https://github.com/airbytehq/airbyte/pull/5501)   | Release Trello CDK Connector                               |
+| Version | Date       | Pull Request                                             | Subject                                                                            |
+|:--------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------|
+| 1.0.3   | 2024-04-02 | [36592](https://github.com/airbytehq/airbyte/pull/36592) | Use python-connector-base image, poetry, and update CDK version                    |
```
Comment (girarda) [inline docs/integrations/sources/trello.md:79]:
```suggestion
| 1.0.3   | 2024-04-10 | [36592](https://github.com/airbytehq/airbyte/pull/36592) | Use python-connector-base image, poetry, and update CDK version                    |
```

File: airbyte-integrations/connectors/source-trello/pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+version = "1.0.3"
+name = "source-trello"
+description = "Source implementation for Trello."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/trello"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-trello/pyproject.toml:15]:
```suggestion
[tool.poetry.packages]
```
Not sure why we have double brackets there, was this autogenerated by the migration command?

File: airbyte-integrations/connectors/source-zenloop/pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+version = "0.1.11"
+name = "source-zenloop"
+description = "Source implementation for Zenloop."
+authors = [ "Alexander Batoulis <alexander.batoulis@hometogo.com>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/zenloop"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-zenloop/pyproject.toml:15]: Same comment about double brackets

File: airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py [hunk 6]
```diff
@@ -32,4 +32,4 @@ def stream_slices(self) -> Iterable[StreamSlice]:
         else:
             for parent_stream_config in self.parent_stream_configs:
                 stream_state_field = parent_stream_config.partition_field.eval(self.config)
-                yield {stream_state_field: custom_stream_state_value, "parent_slice": {}}
+                yield StreamSlice(partition={stream_state_field: custom_stream_state_value, "parent_slice": {}}, cursor_slice={})
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py:35]: very nit: I find it a bit awkward that when reviewing this PR we have no actual clue that the CDK version was updated which likely leads to this change.

File: docs/integrations/sources/instatus.md [hunk 7]
```diff
@@ -55,6 +55,8 @@ The Instatus source connector supports the following [sync modes](https://docs.a
 
 ## Changelog
 
-| Version | Date       | Pull Request                                             | Subject                    |
-|:--------|:-----------|:---------------------------------------------------------|:---------------------------|
-| 0.1.0   | 2023-04-01 | [21008](https://github.com/airbytehq/airbyte/pull/21008) | Initial (alpha) release    |
\ No newline at end of file
+| Version | Date       | Pull Request                                             | Subject                 |
+|:--------|:-----------|:---------------------------------------------------------|:------------------------|
+| 0.1.2 | 2024-03-28 | [0](https://github.com/airbytehq/airbyte/pull/0) | Base image migration: remove Dockerfile and use the python-connector-base image |
```
Comment (alafanechere) [inline docs/integrations/sources/instatus.md:60]:
```suggestion
| 0.1.2 | 2024-03-28 | [36592](https://github.com/airbytehq/airbyte/pull/36592) | Base image migration: Use the python-connector-base image  and poetry|
```

File: docs/integrations/sources/instatus.md
Comment (alafanechere) [inline docs/integrations/sources/instatus.md:1]: Not sure why the instatus doc that got updated 🤔

**Review by alafanechere — CHANGES_REQUESTED** [review]
Minor change request: the `instatus.md` file was updated but it is not touched by this PR.
Nit: I also think it'd be a better practice to split this into two PR to easily revert changes at the connector level if necessary.

**Comment by girarda** [comment]
I'll split into two PRs as recommended

File: airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py (hunk 6 above)
Comment (girarda) [reply airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py:35]: what information would make it easier to review?

Comment (alafanechere) [reply airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py:35]: Nothing, it's just that `^0` is not very explicit, but it makes sense to use it.

**Pull request closed without merging**